build.gradle

```diff
@@ -0,0 +1,132 @@
+plugins {
+  id 'java-library'
+  id 'maven-publish'
+  id 'signing'
+  id "com.diffplug.spotless" version "6.13.0"
+}
+
+group = 'cloud.eppo'
+version = '3.0.0-SNAPSHOT'
+ext.isReleaseVersion = !version.endsWith("SNAPSHOT")
+
+import org.apache.tools.ant.filters.ReplaceTokens
+
+processResources {
+  with copySpec {
+    from 'src/main/filteredResources'
+    filter(ReplaceTokens, tokens: [version: project.properties.version])
+  }
+}
+
+repositories {
+  mavenCentral()
+  mavenLocal()
+}
+
+dependencies {
+  implementation 'cloud.eppo:sdk-common-jvm:0.0.1-SNAPSHOT'
+  implementation 'com.github.zafarkhaja:java-semver:0.10.2'
+  implementation 'com.fasterxml.jackson.core:jackson-databind:2.17.1'
+  implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.17.1'
+  implementation 'com.fasterxml.jackson.module:jackson-module-parameter-names:2.17.1'
```

Comment:
Future PR: Is there a way to get everything to use what Android has natively (`org.json`) but only additionally include it for Java?

Reply:
yup that's possible!